rules/terraform_locals_order.go

```diff
@@ -0,0 +1,124 @@
+package rules
+
+import (
+	"fmt"
+	"github.com/hashicorp/hcl/v2"
+	"sort"
+	"strings"
+
+	"github.com/hashicorp/hcl/v2/hclsyntax"
+	"github.com/hashicorp/hcl/v2/hclwrite"
+	"github.com/terraform-linters/tflint-plugin-sdk/tflint"
+	"github.com/terraform-linters/tflint-ruleset-terraform/project"
+)
+
+// TerraformLocalsOrderRule checks whether comments use the preferred syntax
+type TerraformLocalsOrderRule struct {
+	tflint.DefaultRule
+}
```

Comment:
The rule must be added to the presets.
https://github.com/terraform-linters/tflint-ruleset-terraform/blob/6bee33cc53dc74b9917f4af5591bbbc08e1e577f/rules/preset.go